ethcore/src/snapshot/service.rs

```diff
@@ -622,7 +621,8 @@ impl Service {
 
 							match is_done {
 								true => {
-									db.flush().map_err(UtilError::from)?;
+									// TODO: flush also blooms?
```

Comment:
As I understand the `API` doesn't provide `flushing` but it does `flush` everytime new blooms are inserted?!

If so remove the comment alternatively change the API!

Reply:
that's an obsolete comment ;)
Yes, this db doesn't provide flushing, cause writes are nonatomic and happen immediately when someone calls insert